parsec/core/gui/forms/key_widget.ui

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
New .ui files should be added to `parsec-gui.pro`

Reply:
fixed